crates/red_knot_python_semantic/resources/mdtest/narrow/truthiness.md

```diff
@@ -107,6 +107,7 @@ more accurately.
 def flag() -> bool:
     return True
 
+# TODO: Not sure if this is correct but we're trying to convert `int` to a bool here? Is this intentional.
```

Comment:
not sure I understand the TODO comment here?

Reply:
Oh, this is outdated. That was a comment from before you helped me fix the `bool` implementation.